etl/steps/data/meadow/artificial_intelligence/2023-06-08/yougov_end_of_humanity.py

```diff
@@ -0,0 +1,309 @@
+        column_names (list): The list of column names for the resulting DataFrame.
+        question (str): The question to extract.
+
+    Returns:
+        pd.DataFrame: The extracted data as a DataFrame.
+    """
+    table = shared.read_table_from_pdf(snap.path, 1)
+
+    # Select relevant entries (pdf file reads all lines into 1 messy dataframe; lines starting from 34 are the relevant entries)
+    question2 = table.iloc[34:]
+
+    # Remove the '%' symbol from the selected columns
+    question2 = question2.replace("%", "", regex=True)
+
+    # Select the relevant columns (firs column with options and last five are survey responses) for the DataFrame and convert them to numeric
+    selected_columns = pd.DataFrame(question2.iloc[:, [0] + list(range(-5, 0))])
+    selected_columns.loc[:, selected_columns.columns[-5:]] = selected_columns.loc[
+        :, selected_columns.columns[-5:]
+    ].apply(pd.to_numeric)
+
+    # Drop completely empty rows
+    selected_columns = selected_columns.dropna(axis=0)
+
+    # Reset the index
+    selected_columns.reset_index(drop=True, inplace=True)
+
+    # Rename the columns with more descriptive names
+    selected_columns.columns = [
+        "options",
+        "Very likely",
+        "Somewhat likely",
+        "Not very likely",
+        "Impossible",
+        "Not sure",
+    ]
+
+    # Transpose the DataFrame to have 'options' as the index
+    transposed_df = selected_columns.T
+
+    # Remove the first row (previous column names) and reset the index
+    transposed_df = transposed_df.iloc[1:]
+    transposed_df.reset_index(inplace=True)
+
+    # Rename the columns using the provided column names and question
+    transposed_df.columns = column_names
+    for col in transposed_df.columns[1:]:
+        new_col = question + " " + col
+        transposed_df.rename(columns={col: new_col}, inplace=True)
+
+    return transposed_df
+
+
+def question3_df(snap: Snapshot, column_names: list, question: str) -> pd.DataFrame:
+    """
+    Extract the data for the third question -
+    "How concerned, if at all, are you about the possibility that the following will cause the end of the human race on Earth?",
+    from the second page of a PDF, clean it up and convert it into a DataFrame.
+
+    Args:
+        pdf_path (str): The path to the PDF file.
+        column_names (list): The list of column names for the resulting DataFrame.
+        question (str): The question to extract.
+
+    Returns:
+        pd.DataFrame: The extracted data as a DataFrame.
+    """
+    table = shared.read_table_from_pdf(snap.path, 2)
+    # Skip the first row
+    question3 = table.iloc[1:]
+
+    # Set the column names from the 18th row as the header
+    question3.columns = question3.iloc[0]
+    question3.columns = [
+        "options",
+        "Very concerned",
+        "Somewhat concerned",
+        "Not very concerned",
+        "Not at all concerned",
+        "Not Sure",
+        "Considers Impossible",
```

Comment:
Capitalization is a bit inconsistent here. I'd suggest making capital only the first letter of the sentence.